<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstWindow</class>
 <widget class="QDialog" name="FirstWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1921</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="pie_chart_hbox">
    <item>
     <layout class="QVBoxLayout" name="pie_vbox_1">
      <item>
       <widget class="QLabel" name="jokai_pie_label">
        <property name="text">
         <string>Jókai Mór összes feldolgozott művében található szófajok eloszlása</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="pie_vbox_2">
      <item>
       <widget class="QLabel" name="moricz_pie_label">
        <property name="text">
         <string> Móricz Zsigmond összes feldolgozott művében található szófajok eloszlása</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="next_page_button">
   <property name="geometry">
    <rect>
     <x>1770</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Következő oldal</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>550</y>
     <width>1921</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="pos_vbox"/>
  </widget>
  <widget class="QLabel" name="pos_comp_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>1030</y>
     <width>611</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>A két műben megtalálhaztó szófajok pontos mértéke.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prev_page_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Vissza az előző elemzésekhez</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
